WORKSPACE

```diff
@@ -0,0 +1,65 @@
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+load("@bazel_tools//tools/build_defs/repo:jvm.bzl", "jvm_maven_import_external")
+
+android_sdk_repository(
```

Comment:
How about if we add a documentation link for `android_sdk_repository` like below mentioned?
```
# Android SDK configuration. For more details, see:
# https://docs.bazel.build/versions/master/be/android.html#android_sdk_repository
```

Reply:
Done!